pkgs/development/python-modules/fs/default.nix

```diff
@@ -46,6 +47,7 @@ buildPythonPackage rec {
     mock
     psutil
     pytestCheckHook
+    glibcLocales
```

Comment:
```suggestion
  ]
  ++ lib.optionals isPypy [
    glibcLocales
```
or add a comment. Otherwise someone will remove this again.

Reply:
Done!